.circleci/config.yml

```diff
@@ -7,6 +7,34 @@
 #
 version: 2
 jobs:
+  test:
+    docker:
+      - image: circleci/python:2.7.1
+    steps:
+      - checkout
+      - run:
+          # can this be saved? The python integration tests generally don't change.
```

Comment:
We could use circle's cache, though the challenge is invalidating it when the autopush dependencies change.

Reply:
Yeah, I believe we can probably lock the python integration test for now without much concern. Version updates inevitably will start pulling in py3 dependencies and until we can move these tests to py3, we're probably going to have to not use those.